src/Router/Endpoint.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types=1);
+/**
+ * An endpoint is a HTTP request handler which abstracts away common element
+ * of different LORIS API endpoints.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Dave MacFarlane <dave.macfarlane@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+namespace LORIS\Router;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Server\RequestHandlerInterface;
+use \Psr\Http\Message\ResponseInterface;
+
+/**
+ * An abstract class for common concerns of different API endpoints.
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Dave MacFarlane <dave.macfarlane@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+abstract class Endpoint implements RequestHandlerInterface
+{
+    /**
+     * Return an array of valid HTTP methods for this endpoint
+     *
+     * @return string[] Valid versions
```

Comment:
```suggestion
     * @return string[]
```

Doesn't seem to have anything to do with versions..